tests/src/test_cases/widgets/test_msgbox.c

```diff
@@ -44,7 +44,7 @@ void test_msgbox_creation_successful_with_close_button(void)
 
     TEST_ASSERT_NOT_NULL(msgbox);
 
-    TEST_ASSERT_EQUAL_SCREENSHOT("widgets/msgbox_ok_with_close_btn.png");
+    TEST_ASSERT_EQUAL_SCREENSHOT("ref_imgs/widgets/msgbox_ok_with_close_btn.png");
```

Comment:
"ref_imgs/" is appended automatically [here](https://github.com/lvgl/lvgl/blob/master/tests/unity/unity_support.c#L141) so the original path should be correct.